tests/test_backend.py

```diff
@@ -1,6 +1,6 @@
 import threading
 
-import mock
+from unittest import mock
```

Comment:
Since we're switching from a third party lib to stdlib here, the grouping of the import changes. Should run `isort` again as part of this change.